contracts/core/EntryPoint.sol

```diff
@@ -227,7 +240,7 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuardT
             //address(1) is special marker of "signature error"
             require(
                 address(aggregator) != address(1),
-                FailedOp(totalOps + i, "AA96 invalid aggregator")
+                SignatureValidationFailed(address(aggregator))
```

Comment:
I don't think this is correct

Reply:
I think it is ridiculous to add specific revert reason which basically means "you violated the parameters usage". In most places, we used simple revert (with "Error(string)"). Using FailedOp is a mistake, so either the SignatureValidationFailed(aggregator) - or revert back to Error(string)